README.md

```diff
@@ -155,12 +155,14 @@ Please refer to the [docker](https://docs.docker.com/) docs for more information
 5. Faucet some coins to your coordinator wallet. Hit the mine button afterwards so the transaction gets into a block.
 6. Open `http://localhost:8080/channel/` (note: ensure to add the trailing `/` as otherwise nginx will try to redirect the call)
 7. Copy the address of the lnd node and faucet that wallet as described in step 5.
-8. Copy the node id (_pubkey@host:port_) from your coordinator logs and instruct lnd to open a channel with your coordinator. Set a reasonable channel capacity. Note, this capacity will be only inbound for your coordinator.
+8. Open a channel with your coordinator (02dd6abec97f9a748bf76ad502b004ce05d1b2d1f43a9e76bd7d85e767ffb022c9@[coordinator ip]:9045) and set a reasonable channel capacity.
+
+   _Note, if you're on mac or windows you can use `host.docker.internal` as coordinator ip._
```

Comment:
To make things easy to c&p :) 
```suggestion
   _Note, if you're on mac or windows you can use `host.docker.internal` as coordinator ip._
`02dd6abec97f9a748bf76ad502b004ce05d1b2d1f43a9e76bd7d85e767ffb022c9@host.docker.internal:9045`
```

Reply:
I've added this node id directly to the `open_channel.html` that makes it even more easier to copy & paste 😉